R/Action.R

```diff
@@ -0,0 +1,36 @@
+#' R6 Class representing an action
+#'
+#' An action has a type (right, left, up, down or no action).
+Action <- R6::R6Class("Action", #nolint
+  private = list(
+    .type = NULL,
```

Comment:
Why do variable names start with a dot? I assume this is a Tidyverse standard :-)